xblock/core.py

```diff
@@ -687,7 +687,7 @@ class XBlock(Plugin, Blocklike, metaclass=_HasChildrenMetaclass):
 
     # These are dynamically managed by the awful hackery of _HasChildrenMetaclass.
     # We just declare their types here to make static analyzers happy.
-    # Note that children is only defined iff has_children is defined and True.
+    # Note that children is only defined if has_children is defined and True.
```

Comment:
nit: this comment change can be reverted -- [iff](https://www.dictionary.com/browse/iff) is a shorthand term for "if and only if"

Reply:
I didn't know that! Thanks @pomegranited!!
I dropped the commit.